packages/eslint-config-godaddy-react-flow/index.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  extends: [
+    require('./extends')('eslint-config-godaddy'),
+    'plugin:eslint-plugin-react/recommended'
+  ],
+  plugins: ['react', 'flowtype', 'babel'],
+  parser: 'babel-eslint',
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+      experimentalObjectRestSpread: true
+    }
+  },
+  rules: {
+    'react/display-name': 0,
+    'react/jsx-pascal-case': 2,
+    'react/jsx-uses-react': 1,
+    'react/prefer-es6-class': 2,
+    //
+    // Whitespace rules for specific scenarios (e.g. JSX)
+    //
+    'react/jsx-curly-spacing': [2, 'always', {
+      spacing: { objectLiterals: 'never' }
+    }],
+    'jsx-quotes': [2, 'prefer-single'],
```

Comment:
All of these rules should be inherited from `eslint-config-godaddy-react` no? If we add rules to that package I'd like to see them reflected here.